addons/web/static/src/model/relational_model/utils.js

```diff
@@ -54,6 +55,15 @@ export class FetchRecordError extends Error {
     }
 }
 
+export function fetchRecordErrorHandler(env, error, originalError) {
+    if (originalError instanceof FetchRecordError) {
+        env.services.notification.add(originalError.message, { sticky: true, type: "danger" });
+        return true;
+    }
+}
+const errorHandlerRegistry = registry.category("error_handlers");
+errorHandlerRegistry.add("fetchRecordErrorHandler", fetchRecordErrorHandler);
```

Comment:
I don't think this is the right place for this (a utils file should not have side effect). Maybe move this to its own file?